packages/cosmos-client/src/__tests__/CosmosClient-test.ts

```diff
@@ -0,0 +1,67 @@
+const mockRequest = jest.fn();
+jest.mock("@dcos/http-service", () => ({
+  request: mockRequest
+}));
+
+import { marbles } from "rxjs-marbles/jest";
+import { from } from "rxjs";
+import { take } from "rxjs/operators";
+
+import { CosmosClient } from "../CosmosClient";
+
+describe("CosmosClient", () => {
+  let client: CosmosClient;
+  beforeEach(() => {
+    jest.clearAllMocks();
+
+    client = new CosmosClient("/", "package");
```

Comment:
Hi @TattdCodeMonkey,

[The docs](https://docs.mesosphere.com/1.10/deploying-services/package-api/#routes) seem to say that cosmos has 3 `apiPrefixes` (`/service`, `/package`, `/capabilities`).

Doesn't that mean that if we wanted to ask for capabilities, we currently had to instantiate a `new CosmosClient("/", "capabilities")` to then implement `listCapabilities` if we wanted to stick to [the pattern introduced here](https://github.com/dcos/dcos-ui/pull/3578/files#diff-4559883914d0dedebfd6b7d5d8a392e2R25)?
 
So if we wanted to implement `listCapabilities`, wouldn't we either be forced to not use the instance's `apiPrefix` or to make sure it's `/capabilities` just before we kick off the request?

Reply:
Thats a good point. I was following the example from [here](https://github.com/dcos/dcos-ui/blob/master/src/js/events/CosmosPackagesActions.js#L300) but agree we should just include the prefix in the function's so that the client could support the different prefixes,